packages/amplify-app/package.json

```diff
@@ -24,7 +24,7 @@
     "clean": "rimraf ./lib"
   },
   "engines": {
-    "node": ">=10.0.0",
+    "node": ">=12.0.0",
```

Comment:
Did we do an API review for this?

Reply:
It was discussed in an email, but will stay at 10 for a few weeks.